heapster.go

```diff
@@ -35,8 +35,7 @@ import (
 )
 
 var (
-	argPollDuration    = flag.Duration("poll_duration", 10*time.Second, "The frequency at which heapster will poll for stats")
-	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats. Acceptable values are in the range [1 second, 'poll_duration')")
+	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
```

Comment:
Do you mean this logic in [sources/datasource/utils.go](https://github.com/kubernetes/heapster/blob/master/sources/datasource/utils.go#L24)? That's what I'm proposing in https://github.com/kubernetes/heapster/pull/512#issuecomment-137437419. I think it worth to do it in this particular PR.

If you are ok with those changes I'll do it tomorrow.